dart/lib/src/sentry_stack_trace_factory.dart

```diff
@@ -115,7 +108,7 @@ class SentryStackTraceFactory {
       // unparsed      #04 abs 000000723d4b8c3b virt 0000000000071c3b _kDartIsolateSnapshotInstructions+0x66c3b
 
       // we are only interested on the #01, 02... items which contains the 'abs' addresses.
-      final matches = _absRegex.allMatches(member);
+      final matches = _absRegex.allMatches(member!);
```

Comment:
if `member` is nullable, we cant force `!` but instead checking it

Reply:
Good catch